snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/id/assigner/SnomedNamespaceAndModuleAssignerProvider.java

```diff
@@ -29,12 +32,18 @@
 
 	private static final String NAMESPACE_ASSIGNER_EXTENSION = "com.b2international.snowowl.snomed.datastore.snomedNamespaceAndModuleAssigner";
 	
-	private final Map<String, SnomedNamespaceAndModuleAssigner> assigners;
+	private final Logger LOG = LoggerFactory.getLogger(SnomedNamespaceAndModuleAssignerProvider.class);
+	
+	private final Map<String, SnomedNamespaceAndModuleAssigner> assigners = Maps.newHashMap();
 	
 	private SnomedNamespaceAndModuleAssignerProvider() {
-		assigners = Maps.newHashMap();
 		Extensions.getExtensions(NAMESPACE_ASSIGNER_EXTENSION, SnomedNamespaceAndModuleAssigner.class)
-			.forEach(assigner -> assigners.put(assigner.getName(), assigner));
+			.forEach(assigner -> {
+				SnomedNamespaceAndModuleAssigner previousValue = assigners.put(assigner.getName(), assigner);
```

Comment:
This will keep the last assigner in the registry as opposed to the designed first.
Use `containsKey(String)` to check whether you can register the assigner and if not report it as a warning.